### DISCUSSION TITLE
Various artists

### DISCUSSION THREAD

**Discussion by DeathbyVinyl** [enzo1982/freac#462] [Help & Support]
Hello. When I try and rip multi artist albums sometimes I'm successful and sometimes not. When it doesn't work I usually just get a dozen different individual tracks.

When this happens I go into 'Tags' tick the 'Albums' and put 'Various artists' in the 'Artist' line. (If I look in 'Details' the 'Album Artist' is still incorrect and is greyed out to prevent changes but usually shows whatever it had in there at first - today it had 'Compilation')

So I put in 'Various artists' and go through and edit all the tracks. What happens though is that each track reverts back to it's original format.

So, my query is - how to force a disc to be ripped as a Various artists album with a single Album title holding a dozen tracks by individual artists.

Just to note - this is not always the case. Some discs rip perfectly!

Thanks for looking.

Martin

**Comment by znarkd** [comment]
Within the "Tags" tab, select the "Tracks" tab and then the "Basic" sub-tab.  Enter the Artist name into the "Artist" column.  Then select the "Details" sub-tab and enter "Various Artists" within the "Album artist" column.  Do this for each individual song/title in the album.

**Reply by DeathbyVinyl** [reply]
Hello and thanks for the reply. Unfortunately this doesn't work for me. The fields you mention are greyed out and don't accept input. I have solved my problem though - I tried using Audex and found that that works perfectly (although I still use fre:ac for everything else!) Thanks again for the reply. Regards, Martin

**Comment by enzo1982** [comment]
Hi,

Generally, your approach of putting _Various artists_ in the _Tags->Albums->Basic->Artist_ field is correct. This should set all the tracks' artists to _Various artists_. You can then go back to the _Tracks_ tab and enter the individual artists for each track in the same field. The _Details->Album artist_ field for each track should still say _Various artists_ after that.

Then just make sure to use `<albumartist>` instead of `<artist>` in your filename pattern and you should be good to go.

If it doesn't work like this, please make sure you are using the latest version. There were several issues with editing artist/album artist in earlier versions of fre:ac.

I can confirm that the value in the _Tags->Albums->Details->Album artist_ field does not update when changing the value in the _Basic->Artist_ field. That's a bug and will be fixed soon. This should only be a display issue, though, and not affect operation.

Robert
